uber/templates/base.html

```diff
@@ -98,13 +98,17 @@
             });
             $('.focus:first').focus();
             if (window.DISABLE_STRIPE_BUTTONS_ON_CLICK) {
-                $('form.stripe').on('click', 'button', function() {
-                    $('a > .stripe-button-el').unwrap().attr('disabled', true).unwrap()
```

Comment:
This was ultimately the solution.  For anyone interested in why this was tricker than expected:
* The stripe form loads in an iframe, so we can't listen in on anything on the form itself, since we don't have access to the contents of the iframe.
* We can listen for when the iframe closes, but since it closes both on success and cancellation, we can't do anything differently based on that.
* Stripe Checkout has no callbacks or events you can listen to to tell when the form is submitted or canceled.  (You can get this if you don't use the form approach and instead do custom Javascript, which is pretty simple but was a much bigger rewrite than I wanted to do here... we should consider that for the future though.)
* You can't do ``$(form).on('submit', ...)`` because Stripe just calls ``form.submit()`` which bypasses ``onSubmit`` listeners.

The solution here approximates waiting for a form submission.  I'll have to remember this in the future.  Or hopefully I'll forget it and never need this knowledge again :P